### PR TITLE
The Krispy Krunchiest Update

### PR DESCRIPTION
**Pull request by rswiernik**
Come gather 'round people
Wherever you ride
And admit that the bike lanes
Around you are just fine
And accept it that soon
You'll be drenched to the bone
If your chicken to you's worth savin'
Then you better start eatin'
Or you'll bonk like a stone
For the times they are a-changin'

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,6 +38,7 @@ <h2>Map</h2>
 
       <h2>Stops</h2>
       <ol>
+        <li>(Optional) Krispy Krunchy Chicken &amp; 7/11 at Mason St.</li>
```
Comment (skalnik) [inline index.html:41]:
```suggestion
        <li>(Optional) Krispy Krunchy Chicken &amp; 7-Eleven at Mason St.</li>
```

File: index.html [hunk 2]
```diff
@@ -105,6 +106,15 @@ <h2>FAQ</h2>
             after that.
           </p>
         </dd>
+        <dt>What's the food situation?</dt>
+        <dd>
+          <p>
+            For food during the ride, folks stop at Krispy Krunchy Chicken on Mason St. in
+            Fisherman's wharf. There is a 7/11 next door for haribo and refreshments if that's
+            more your speed. The ride ends at Benders usually in time to grab food before the
+            kitchen closes for the night.
```
Comment (skalnik) [inline index.html:115]:
```suggestion
            For food during the ride, folks stop at Krispy Krunchy Chicken on Mason St. in
            Fisherman's wharf. There is a 7-Eleven next door for Haribo and refreshments if that's
            more your speed. The ride ends at Bender's in time to grab food before the kitchen
            closes for the night.
```

**Review by skalnik — APPROVED** [review]
Absolutely incredible work. I think this is mostly good to go but I'm hesitant to add it unless more folks actually hit this regularly rather than just Jordan

kfc-fried-chicken

File: index.html [hunk 3]
```diff
@@ -105,6 +106,15 @@ <h2>FAQ</h2>
             after that.
           </p>
         </dd>
+        <dt>What's the food situation?</dt>
+        <dd>
+          <p>
+            For food during the ride, folks stop at Krispy Krunchy Chicken on Mason St. in
+            Fisherman's wharf. There is a 7/11 next door for haribo and refreshments if that's
+            more your speed. The ride ends at Benders usually in time to grab food before the
```
Comment (Drewzar) [inline index.html]: Should we also be shouting out Overkill Grill?

**Review by Kasbr305 — APPROVED** [review]

**Pull request closed without merging**